server/graphql/v1/mutations/expenses.js

```diff
@@ -348,3 +359,38 @@ export async function payExpense(remoteUser, expenseId, fees = {}) {
 
   return markExpenseAsPaid(expense);
 }
+
+export async function markExpenseAsUnpaid(remoteUser, ExpenseId) {
+  if (!remoteUser) {
+    throw new errors.Unauthorized('You need to be logged in to pay an expense');
+  }
+
+  const expense = await models.Expense.findByPk(ExpenseId, {
+    include: [{ model: models.Collective, as: 'collective' }, { model: models.User, as: 'User' }],
+  });
+
+  if (!expense) {
+    throw new errors.NotFound('No expense transaction found');
+  }
+
+  if (!canMarkExpenseUnpaid(remoteUser, expense)) {
+    throw new errors.Unauthorized("You don't have permission to mark this expense unpaid");
+  }
+
+  if (expense.status !== statuses.PAID) {
+    throw new errors.Unauthorized('Expense has not been paid');
+  }
+
+  if (expense.payoutMethod !== 'other') {
+    throw new errors.Unauthorized('Only expense with other payout method can be marked unpaid');
+  }
+
+  const transaction = await models.Transaction.findOne({
+    where: { ExpenseId },
+    include: [{ model: models.Expense }],
+  });
+  const refundedTransaction = await libPayments.createRefundTransaction(transaction, 0, null, expense.User);
```

Comment:
I'm not sure about the `0`. When hosts mark expenses as paid, they can set a `paymentProcessorFeeInHostCurrency`. If the initial payment processor fee is not 0, then this will take the amount out  of the host budget. But that's tricky.

@alanna When un-paying expenses with `Other` payout method, should we assume that:
1. Payment processor fees are lost in the payout, and we should take them out of the host's budget
2. OR that fees are reimbursed by whatever the payout method and it's fine to mark them as such

My assumption will be that 2. is the valid one, because the reason for marking expenses as unpaid is usually that the payment has failed and no fees is generally taken in those cases. 

But maybe host should be able to set this when marking as unpaid (ex: Click on the button -> Checkbox: Has the payout provider refunded the payment processor fees? -> Yes/No).

Reply:
Let's wait on @alanna's feedback.

Reply:
@Betree What do you think?

<img width="526" alt="Screenshot 2019-10-06 at 9 19 57 AM" src="https://user-images.githubusercontent.com/15707013/66266284-951cd800-e81a-11e9-9ce1-e3f7fe3edf1d.png">

Reply:
That'd be perfect. I would have it checked by default (based on my assumption above).

Let's add the param for it to the query and based on that, pass either `0` or `transaction.paymentProcessorFeeInHostCurrency` based on that.